src/mono/mono.proj

```diff
@@ -417,7 +417,8 @@
       <_MonoRuntimeFilePath Condition="'$(TargetstvOS)' == 'true'">$(MonoObjDir)out\lib\libmonosgen-2.0.dylib</_MonoRuntimeFilePath>
       <_MonoRuntimeFilePath Condition="'$(TargetsAndroid)' == 'true'">$(MonoObjDir)out\lib\libmonosgen-2.0.so</_MonoRuntimeFilePath>
       <_MonoRuntimeFilePath Condition="'$(TargetsBrowser)' == 'true'">$(MonoObjDir)out\lib\libmonosgen-2.0.a</_MonoRuntimeFilePath>
-      <_MonoRuntimeFilePath Condition="'$(_MonoRuntimeFilePath)' == ''">$(MonoObjDir)mono\mini\.libs\libmonosgen-2.0.so</_MonoRuntimeFilePath>
+      <_MonoRuntimeFilePath Condition="'$(_MonoRuntimeFilePath)' == '' and '$(TargetsLinux)' == 'true' and '$(TargetArchitecture)' == 'x64'">$(MonoObjDir)out\lib64\libmonosgen-2.0.so</_MonoRuntimeFilePath>
```

Comment:
Shouldn't we be trying to move more of this to CMake rather than less?

Reply:
I didn't find an easy way to overwrite the `lib` value that cmake uses for the install directory.

Reply:
Why is this needed, are those files different ?

Reply:
not really, but they could be if you add custom install rules. the main thing is it makes us consistent accross platforms.